cli/azd/internal/vsrpc/handler.go

```diff
@@ -38,6 +42,10 @@ func HandlerAction0(f func(context.Context) error) Handler {
 // HandlerAction1 is a helper for creating a Handler from a function that takes one argument and returns an error.
 func HandlerAction1[T1 any](f func(context.Context, T1) error) Handler {
 	return func(ctx context.Context, conn jsonrpc2.Conn, reply jsonrpc2.Replier, req jsonrpc2.Request) error {
+		if err := verifyArity(req, 1); err != nil {
```

Comment:
nit. merely esthetic to my picky eyes...  Can we have constants for these numbers, so we can read:

```go
const arityOne = 1

if err := verifyArity(req, arityOne); err != nil {
```